subsys/fs/zms/zms.c

```diff
@@ -110,6 +112,19 @@ static inline off_t zms_addr_to_offset(struct zms_fs *fs, uint64_t addr)
 	return fs->offset + (fs->sector_size * SECTOR_NUM(addr)) + SECTOR_OFFSET(addr);
 }
 
+/* Helper to round down len to the closest multiple of write_block_size  */
+static inline size_t zms_get_multiple_write_block_size(struct zms_fs *fs, size_t len)
```

Comment:
I think the naming could be made cleared than that. e.g.:
```suggestion
static inline size_t zms_round_down_write_block_size(struct zms_fs *fs, size_t len)
```